docs/internal-documentation/distrib-testing/Ubuntu-build-environment.md

```diff
@@ -0,0 +1,16 @@
+# Ubuntu Build Environment
+
+These are instructions for creating a build environment for Zonemaster
+components based on Perl. This is not meant as instructions for installing
+Zonemaster itself. 
+
+This instruction is for creating it on Ubuntu. See other files for other OSs.
+
+> Should this be Ubuntu 20.04? Do we need to cover more?
+
+> Is git installed by default?
+>
+> Here we need gettext. Anything else?
+>
+> Can we just link to the instruction for Debian?
```

Comment:
I think this should be possible but I haven't checked that the process doesn't differ between Debian and Ubuntu.